analyzer/evmnfts/evm_nfts.go

```diff
@@ -116,12 +116,13 @@ func (p *processor) ProcessItem(ctx context.Context, batch *storage.QueryBatch,
 		staleNFT.Addr,
 		staleNFT.ID,
 		staleNFT.DownloadRound,
-		nftData.MetadataURI,
+		storage.SanitizeStringP(nftData.MetadataURI),
 		nftData.MetadataAccessed,
-		nftData.Metadata,
-		nftData.Name,
-		nftData.Description,
-		nftData.Image,
+		// If SanitizeString can affect JSON validity, we're doomed.
```

Comment:
:(
It does seem low risk.
We could have another wrapper, `ensureValidJSON()`, that returns the input as-is if it's valid JSON, otherwise logs a warning and returns `{}` or `null`. Your call on whether that's warranted. I think you already made the call with this comment :)